server/schedule/filter/metrics.go

```diff
@@ -22,7 +22,7 @@ var (
 			Subsystem: "schedule",
 			Name:      "filter",
 			Help:      "Counter of the filter",
-		}, []string{"action", "address", "store", "type"})
+		}, []string{"action", "address", "store", "acton", "type"})
```

Comment:
Maybe "act-on" is better?

Reply:
"acton" is too similar to "action". Is there any reason for using it instead of "act-on"?